node-fs-readfile/cat.js

```diff
@@ -0,0 +1,15 @@
+import { readFile } from 'node:fs/promises';
+
+async function cat(args) {
+  try {
+    for (let i = 2; i < args.length; i++) {
+      const filePath = new URL(args[i], import.meta.url);
```

Comment:
Although the Node documentation may look like you need import.meta.url, etc., it actually isn't necessary. By default, readFile will look in the current directory for the file.

So, you can write this much more succinctly as:
`const promises = arrayOfFiles.map((filePath) => readFile(filePath, 'utf8'));`

Note that as a shortcut, readFile allows you to send the encoding as a string in the second parameter.